pysrc/kerberos.py

```diff
@@ -193,7 +193,36 @@ def authGSSClientInquireCred(context):
 
 
 
-def authGSSClientStep(context, challenge):
+def buildChannelBindingsStruct(**kwargs):
+    """
+    Builds a gss_channel_bindings_struct which can be used to pass onto
+    L{authGSSClientStep} to bind onto the auth. Details on Channel Bindings
+    can be foud at https://tools.ietf.org/html/rfc5929. More details on the
+    struct can be found at
+    https://docs.oracle.com/cd/E19455-01/806-3814/overview-52/index.html
+
+    @param initiator_addrtype: Optional integer used to set the
+        initiator_addrtype, defaults to GSS_C_AF_UNSPEC if not set
```

Comment:
I think you'll also want to add module constant's for GSS_C_AF_UNSPEC and whatever other values are valid for *addrtype, similar to the definitions for things like AUTH_GSS_COMPLETE, GSS_C_DELEG_FLAG here:

https://github.com/apple/ccs-pykerberos/blob/PyKerberos-1.2.5/src/kerberos.c#L802-L835